index.html

```diff
@@ -17,7 +24,7 @@
     <!-- NAVIGATION SECTION -->
     <header>
         <nav class="flex align-center justify-around gap-m">
-            <img src="./imgs/logo1.png" alt="website logo reading: ZanyMen" />
+            <img src="./imgs/logo.png" alt="website logo reading: ZanyMen" />
```

Comment:
looks like this logo is bigger. The navbar now overflows a bit to the sides when you view it in any of the mobile sizes. I'm not sure if it's because of this one, but it's worth checking if it can be solved by adjusting the size of the logo.